masks/mask_class/NfPR.py

```diff
@@ -0,0 +1,283 @@
+import numpy as np
```

Comment:
all over these functions already exist in [waveprop.util](https://github.com/ebezzam/waveprop/blob/master/waveprop/util.py) so we should import from waveprop, as done in [here](https://github.com/LCAV/LenslessPiCam/blob/4a84625fa2d4081c34294a9c34b3b3af835fb47c/scripts/sim/single_file.py#L22) and add `waveprop` to requirements, rather than duplicating the code